components/raftstore/src/store/snap.rs

```diff
@@ -1013,6 +1013,26 @@ impl Snapshot {
             delete_file_if_exist(&self.meta_file.tmp_path).unwrap();
         }
     }
+
+    fn new_for_tablet_snapshot<T: Into<PathBuf>>(
```

Comment:
please add comments that this function is for v2 compatible.

Reply:
Done